advanced/sr_gui_cyberglove_calibrator/CMakeLists.txt

```diff
@@ -1,10 +1,10 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(sr_gui_cyberglove_calibrator)
-find_package(catkin REQUIRED COMPONENTS rospy rqt_gui rqt_gui_py std_msgs std_srvs diagnostic_msgs sr_robot_msgs sr_visualization_icons)
+find_package(catkin REQUIRED COMPONENTS rospy rqt_gui rqt_gui_py std_msgs std_srvs diagnostic_msgs sr_robot_msgs sr_visualization_icons python-scipy)
```

Comment:
Indeed, well spotted, without this system install, we get
ImportError: No module named scipy.optimize, I did not look good enough when trying to find the usage.